tests/format/js/method-chain/13018.js

```diff
@@ -1,3 +1,6 @@
 foo(_a).bar().leet();
 foo(-a).bar().leet();
 foo(+a).bar().leet();
+foo(~a).bar().leet();
+foo(++a).bar().leet();
```

Comment:
I assume we are going to treat `a++`  the same, please add test for it.

Reply:
I added tests for `a++` and `a--`.